beamer/models/claim.py

```diff
@@ -53,14 +57,19 @@ def request_id(self) -> RequestId:
     def claimer(self) -> Address:
         return self._latest_claim_made.claimer
 
-    @property
-    def challenger(self) -> Address:
-        return self._latest_claim_made.challenger
-
     @property
     def termination(self) -> Termination:
         return self._latest_claim_made.termination
 
+    def get_challenger_stake(self, challenger: Address) -> int:
+        if challenger in self._challenger_stakes:
+            return self._challenger_stakes[challenger]
+        return 0
+
+    def add_challenger_stake(self, challenger: Address, amount: int) -> None:
```

Comment:
This could be private, no?

Reply:
it could be private because we don't use it right now outside. But it could be the case in the future and this would be totally fine, and I'm afraid that we forget why we chose it to be private.
I mean it is simply not used outside of the class but it's totally fine to do so.